src/game/game_init.c

```diff
@@ -28,9 +28,12 @@
 #include "puppyprint.h"
 #include "puppycam2.h"
 #include "debug_box.h"
-#include "vc_check.h"
 #include "vc_ultra.h"
 #include "profiling.h"
+#include "emutest.h"
+
+// Emulators that the Instant Input patch should not be applied to
+#define INSTANT_INPUT_BLACKLIST (EMU_CONSOLE | EMU_WIIVC | EMU_ARES | EMU_SIMPLE64)
```

Comment:
imo this should be defined in emutest.h since that can be served as feature configuration based on emulator

Reply:
yeah thats fair. resolving